README.md

```diff
@@ -31,20 +31,25 @@ cmake --build . --config Release
 
 If everything went OK, `bin\Release\rwkv.dll` file should appear.
 
-### 2. Download an RWKV model from [Hugging Face](https://huggingface.co/BlinkDL) and convert it into `ggml` format
-
+### 2. Download an RWKV model from [Hugging Face](https://huggingface.co/BlinkDL) like [this one](https://huggingface.co/BlinkDL/rwkv-4-pile-169m/blob/main/RWKV-4b-Pile-171M-20230202-7922.pth) and convert it into `ggml` format
```

Comment:
[As RWKV author says](https://huggingface.co/BlinkDL/rwkv-4-pile-169m),

>Use RWKV-4 models (NOT RWKV-4a, NOT RWKV-4b) unless you know what you are doing. 

`4a` and `4b` models are experiments containing new model parts, and will not work good with `rwkv.cpp`. Please change links and file names below to use regular `4` model, not `4b` :)